### ISSUE TITLE
Port caffemodel to hdf5

### ISSUE THREAD

**Issue by vinodlahire** [rcmalli/keras-vggface#28]
@rcmalli original VGGFACE1 .caffemodel is around 553MB size ; can you refer links / method that you convert these models to ~90 MB of hdf5 without loss of accuracy ? I am using your ported model for face verification task ; now i want to train my own model in caffe and port it to keras.

**Comment by Lahirevinod** [comment]
figured out ; wights are without top so having less memory footprint... my bad
